terraform/modules/data-share-service/cloudwatch.tf

```
@@ -41,22 +41,40 @@ module "metrics_dashboard" {
       period = local.metric_period,
       stat   = "Sum",
       metrics = [
-        { name = "API_CALLS.PublishEvent.count", dimensions = { success = true }, attributes = { label = "PublishEvent" } },
+        { name       = "API_CALLS.PublishEvent.count", dimensions = { success = true },
+          attributes = { label = "PublishEvent" }
+        },
```

can we multiline this a bit more consistently pls :) 
```suggestion
        {
          name       = "API_CALLS.PublishEvent.count", 
          dimensions = { success = true },
          attributes = { label = "PublishEvent" }
        },
```